CHANGES.md

```diff
@@ -1,3 +1,10 @@
+### unreleased
+
+#### Changes
+
+  + The default for `doc-comments` is changed to `after` (#1335) (Jules Aguillon)
+    This revert a change introduced in 0.14.0.
```

Comment:
Can you mention #1325 here so we can easily jump back and forth between the PRs ? thanks

Reply:
The change is from https://github.com/ocaml-ppx/ocamlformat/pull/1012. The change to the `ocamlformat` profile is not reverted by this PR.